Makefile

```diff
@@ -197,11 +203,15 @@ endif
 
 
 ifeq ($(detected_OS),Darwin)
-  ifeq ("$(shell sysctl -nq hw.optional.arm64)","1")
-    # Building on M1 is still not supported, so in the meantime we crosscompile to amd64
-    DOTHERSIDE_CMAKE_PARAMS += -DCMAKE_OSX_ARCHITECTURES=x86_64
+ ifeq ("$(shell sysctl -nq hw.optional.arm64)","1")
```

Comment:
@alexjba can you please clarify when we need this case of cross-compiling x86 on arm for MacOS?

Reply:
We don't have support on CI machines for arm builds. This check is here so that we don't break the CI until we can pack an universal build.
My plan is to get this approved by dev and QA. And once we're confident we can release the universal build to work on CI and add support for universal builds.